source/js/gallery.binding.js

```diff
@@ -1,14 +1,22 @@
 /**
  * @file
- * Gallery component binding.
+ * Javascript binding of baguettebox.min.js.
```

Comment:
Prefer the old comment.
This is not binding of baguettebox, this is a generic binding of **a** lightbox implementation.

Reply:
Reverted this.